o/oas-kit/oas-kit.sh

```diff
@@ -0,0 +1,61 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: oas-kit
+# Version	: v5.0.0
+# Source repo	: https://github.com/Mermade/oas-kit
+# Tested on	: UBI 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: BulkPackageSearch Automation {maintainer}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=oas-kit
+PACKAGE_VERSION=v5.0.0
```

Comment:
can we make this configurable - allow user to pass it as an argument to the script?

Reply:
Updated